test/report/report.test.ts

```diff
@@ -28,7 +28,7 @@ describe("report", () => {
         });
 
         describe("exportResult", () => {
-            it("should sanitized showFilters config", () => {
+            it("should sanitized afm filter in export config", () => {
```

Comment:
typo `sanitized` -> `sanitize`

Reply:
Fixed in 6d956b9f1cb0f7990ec1bda2949c21280d484364.